source/Renderer/shaders/displaymapping.glsl

```diff
@@ -0,0 +1,53 @@
+// KHR_displaymapping_pq
+
+uniform float u_ApertureFactor;  // Calculated by using max light intensity value of the scene
```

Comment:
I would propose adding:
uniform float u_rangeExtension; //Set to 46.42 for sRGB buffer (SDR) and 59.5208 for ST2084 PQ buffer (HDR)